src/pug/sidebar.pug

```diff
@@ -1,2 +1,4 @@
+script
+    | console.log('Deprecation warning: Please update your usage of Chrome.')
```

Comment:
`console.warn` is much better. Logs are usually overlooked.

Reply:
Will do.